clients/dos/src/main/kotlin/DosModel.kt

```diff
@@ -46,6 +46,15 @@ data class UploadUrlResponseBody(
     val message: String? = null
 )
 
+/** Information needed to create a Package row in DOS database */
+@Serializable
+data class PackageInfo(
```

Comment:
Is the use of this class limited (in future) only to the request for creating a scan job? If yes, then consider moving the class (as inner class) into the job request and name it just package.

Reply:
This class is used also in the scan results request in the next commit